shared-helpers/__tests__/pdfs.test.ts

```diff
@@ -5,35 +5,46 @@ import { cloudinaryPdfFromId, pdfUrlFromListingEvents } from "../src/utilities/p
 afterEach(cleanup)
 
 describe("pdfs helpers", () => {
+  const OLD_ENV = process.env
```

Comment:
Some steps below, like lines 20 and 27, mutate the `process.env` obj.  Keeping the original state at the start of the tests makes it possible to reset in between each one (line 10).  The `process.env.CLOUDINARY_CLOUD_NAME` var is (tragically) used all over the place across our services, so while I think it might be possible to just set that explicitly at the beginning and reuse it, it seems a bit more "clean" to reset before each test IMO.

Reply:
Yeah as Chris explained it's to return back to the original state after making changes. The beforeEach() part is unnecessary though since the value set is the same for all the tests so I removed that part and just set the cloud name at the start (otherwise it's undefined).

Reply:
gotcha, thanks!